src/main.cpp

```diff
@@ -401,12 +401,12 @@ static void print_allowed_options(const OptionList &allowed_options)
 static void print_version()
 {
 #ifdef SERVER
-	dstream << "minetestserver " << minetest_version_hash << std::endl;
+	dstream << PROJECT_NAME "-server " << g_version_hash << std::endl;
```

Comment:
Intentional, although on second thought there should be a space instead of a dash.